MODULE.bazel.lock

```diff
@@ -5774,12 +5774,12 @@
         "bzlTransitiveDigest": "BPi/jkbGe4t+CftMYaXRe/Clkd62vRQhWpqTKbsi77E=",
         "usagesDigest": "iKmmEQX8xqicrHiRHLqXwjwu7pCrLkIAFXGNdc4Aveo=",
         "recordedFileInputs": {
-          "@@rules_rust~~rust_host_tools~rust_host_tools//bin/rustc": "241027b94ad67beb36d0356f7cc05180daaa9966b2958cfba1d4de089d2e521c",
+          "@@rules_rust~~rust_host_tools~rust_host_tools//bin/rustc": "b953c69e6f682bf55e39bdd55bdcbd3e1cc0cbdf9dfda24b9c6478ee2c38359b",
```

Comment:
Note: I've no idea why this line has changed. Please review accordingly.

Reply:
It's a known issue, I will fixup